content/10.diagnostics.md

```diff
@@ -51,15 +71,17 @@ For example, PCR-based test can be highly sensitive, but in high-throughput sett
 3. Type of specimen, as it is not clear which clinical samples are best to detect the virus [@doi:10.1038/d41587-020-00002-2].
 4. Expensive machinery, which might be present in major hospitals and/or diagnostic centers but is often not available to smaller facilities [@doi:10.1126/science.abb8400].
 5. Timing of the test, which might take up to 4 days to give results [@doi:10.1126/science.abb8400].
-
-False positives are a concern with this technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome, resulting in a loss of specificity.
-
+6. The availability of supplies for testing, including swabs and testing media, has been limited [@doi:10.1128/JCM.00512-20].
+7. Because the guide RNA can recognize other interspersed sequences on the patient’s genome, false positives and a loss of specificity can occur
+8. False positives are a concern with this technique, as the guide RNA can recognize other interspersed sequences on the patient’s genome, resulting in a loss of specificity.
 Similarly, in tests that use CRISPR, false positives can occur due to the specificity of the technique, as the guide RNA can recognise other interspersed sequences on the patient’s genome.
+As noted above, false negatives are a significant concern for several reason.
+Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20]
```

Comment:
```suggestion
Importantly, clinical reports indicate that it is imperative to exercise caution when interpreting the results of molecular tests for SARS-CoV-2 because negative results do not necessarily mean a patient is virus-free [@doi:10.1128/JCM.00297-20].
```